haystack/components/evaluators/answer_f1.py

```diff
@@ -0,0 +1,69 @@
+from typing import Dict, List
+
+from haystack.core.component import component
+
+
+@component
+class AnswerF1Evaluator:
+    """
+    Evaluator that calculates the F1 score of the matches between the predicted and the ground truth answers.
```

Comment:
```suggestion
    Evaluator that calculates the average F1 score of the matches between the predicted and the ground truth answers.
```
More accurate, I think.